static/partials/home.html

```diff
@@ -103,25 +121,39 @@
         <div class="row">
             <div class="col-lg-6 col-md-6 col-sm-6">
                 <h2>Warunki uczestnictwa:</h2>
-                <p>W warsztacie może wziąć udział każdy, niezależnie od płci, wieku (16+), wykształcenia, czy wykonywanego zawodu. Konieczne są:</p>
+                <p>W warsztacie może wziąć udział każdy, niezależnie od płci,
+                    wieku (16+), wykształcenia, czy wykonywanego zawodu.
+                    Konieczne są:</p>
                 <ul>
                     <li>Własny laptop</li>
-                    <li>Zainstalowane środowisko <a href="https://www.youtube.com/watch?v=0d6znPZb3PQ&t=3s">(instrukcja)</a></li>
-                    <li>Przejście <a href="https://pyladiesstart.pl/#/register">rekrutacji</a></li>
-                    <li>Podstawowa znajomość języka angielskiego. <i>(Warsztaty będą prowadzone w języku polskim, jednakże dla opanowania składni języka Python znajomość języka angielskiego będzie niewątpliwie pomocna)</i>
+                    <li>Zainstalowane środowisko <a
+                            href="https://www.youtube.com/watch?v=0d6znPZb3PQ&t=3s">(instrukcja)</a>
+                    </li>
+                    <li>Przejście <a
+                            href="https://pyladiesstart.pl/#/register">rekrutacji</a>
+                    </li>
+                    <li>Podstawowa znajomość języka angielskiego. <i>(Warsztaty
+                        będą prowadzone w języku polskim, jednakże dla
+                        opanowania składni języka Python znajomość języka
+                        angielskiego będzie niewątpliwie pomocna)</i>
                 </ul>
-                <p>Dodatkowo prosimy o zaopatrzenie się w jedzenie we własnym zakresie, którym będzicie się mogli i chcieli podzielić z innymi uczestnikami. Zapewniamy jedynie ciepłe i chłodne napoje.</p>
-                <p>Potrzebujemy także użyczenia listew zasilających, najchętniej 5- lub 6-gniazdowych. Prosimy o Wasze wsparcie i deklarację w tej sprawie podczas rejestracji.</p>
+                <p>Dodatkowo prosimy o zaopatrzenie się w jedzenie we własnym
+                    zakresie, którym będzicie się mogli i chcieli podzielić z
+                    innymi uczestnikami. Zapewniamy jedynie ciepłe i chłodne
+                    napoje.</p>
```

Comment:
`Dodatkowo prosimy o zaopatrzenie się we własnym
                     zakresie w jedzenie, którym będziecie mogli się podzielić z
                     innymi uczestnikami. Zapewniamy jedynie ciepłe i chłodne
                     napoje`